src/TransientCleaner.php

```diff
@@ -0,0 +1,260 @@
+<?php
+/**
+ * Delete WordPress transients on content save, based on user-input configuration.
+ *
+ * @package    studiometa/wp-toolkit
+ * @author     Studio Meta <agence@studiometa.fr>
+ * @copyright  2020 Studio Meta
+ * @license    https://opensource.org/licenses/MIT
+ * @since      1.0.0
+ * @version    1.0.0
+ */
+
+namespace Studiometa\WPToolkit;
+
+/**
+ * TransientCleaner class.
+ */
+class TransientCleaner {
+	const PREFIX                   = 'wp_transient_cleaner_';
```

Comment:
**Nitpick** : should the default prefix contain `wp_toolkit` ? In order to clearly identity the source from the transient ?